buildSrc/build.gradle.kts

```diff
@@ -9,7 +9,7 @@ repositories {
 
 dependencies {
     implementation("com.moowork.gradle:gradle-node-plugin:1.3.1")
-    implementation("org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.31")
+    implementation("org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.40")
```

Comment:
Can we reuse kotlin_version somehow?

Reply:
Done in https://github.com/badoo/Reaktive/pull/126